x11-misc/xembedsniproxy/xembedsniproxy-5.12.0.ebuild

```diff
@@ -6,7 +6,7 @@ EAPI=6
 inherit cmake-utils
 
 DESCRIPTION="A program to use legacy Xembed tray icons with SNI-only trays"
-HOMEPAGE="https://kde.org"
+HOMEPAGE="https://github.com/KDE/plasma-workspace/tree/master/xembed-sni-proxy"
```

Comment:
That's a git subdir on a mirror of the project, do you think that's appropriate as homepage?

Reply:
Hm, you're right about it being a mirror, thanks. Though unfortunately, no other page (I found) refers to the program, and I was convinced in IRC that a git repo subdir IS more appropriate than kde.org.